cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample/terraform/main.tf

```diff
@@ -61,6 +65,41 @@ resource "azurerm_cosmosdb_account" "application" {
   }
 }
 
+# Used to get object_id
+data "azurerm_client_config" "current" {
+}
+
+resource "random_string" "random" {
+  length = 5
+  min_lower = 5
+  special = false
```

Comment:
do we really need this, is the role definition unique to a cosmos instance? if so, how about just use cosmosdb-sql-role-definition